src/styles/Home.module.css

```diff
@@ -1,278 +0,0 @@
-.main {
```

Comment:
@ibimina Seeing as this file has nothing in it (as I saw locally) why are you still including it in this PR?

Reply:
I have deleted the empty stylesheet